guides/common/assembly_administering-hosts.adoc

```diff
@@ -24,7 +24,10 @@ include::modules/proc_changing-the-environment-of-a-host.adoc[leveloffset=+1]
 
 include::modules/proc_changing-the-managed-status-of-a-host.adoc[leveloffset=+1]
 
-include::modules/proc_enabling-tracer-on-a-host.adoc[leveloffset=+1]
+:parent-context: {context}
+:context: administering-hosts
+include::modules/proc_configuring-tracer-on-a-host.adoc[leveloffset=+1]
+:context: {parent-context}
```

Comment:
```suggestion
:context: {parent-context}
:!parent-context:
```


alternative idea: do not make any changes to "guides/common/assembly_administering-hosts.adoc" and instead add a ".Next steps" section to "guides/common/modules/proc_registering-a-host.adoc" that links to "Configuring Tracer on a host" so that we do not include it twice in the same guide.